packages/frontend/src/components/Buttons/AccountButton.tsx

```diff
@@ -1,14 +1,27 @@
 import { shortenAddress, useEthers } from '@usedapp/core'
+import { useCallback } from 'react'
+import { removeWalletLinkStorage } from 'src/utils/removeWalletLinkStorage'
 import styled from 'styled-components'
 
 import { Button } from './Button'
 import { ConnectWalletButton } from './ConnectWalletButton'
 
 export const AccountButton = () => {
-  const { account } = useEthers()
+  const { account, deactivate } = useEthers()
+
+  const disconnect = useCallback(async () => {
+    localStorage.removeItem('walletconnect')
+    removeWalletLinkStorage()
+    deactivate()
+  }, [deactivate])
 
   return account ? (
-    <ConnectedButton view="secondary">{shortenAddress(account)}</ConnectedButton>
+    <>
+      <ConnectedButton view="secondary" onClick={disconnect}>
+        Disconnect
```

Comment:
Let's remove disconnect button from here. It is added in my PR with account details modal